src/core_modules/capture-core/components/WidgetEventSchedule/WidgetEventSchedule.container.js

```diff
@@ -62,6 +62,18 @@ export const WidgetEventSchedule = ({
             notes: [],
         }] });
     };
+    const onAddComment = async (comment) => {
+        await mutate({ events: [{
```

Comment:
Don't know if you changed this later, but when adding a comment we should preserve it in memory and send it to the api when you are actually scheduling the event.

Reply:
added to redux in branch DHIS2-11864, [commit](https://github.com/dhis2/capture-app/pull/2216/commits/4625880255c215e602759c0457c5045f591be446)